partiql-lang/src/test/kotlin/org/partiql/lang/eval/EvaluatingCompilerGraphMatchTests.kt

```diff
@@ -0,0 +1,24 @@
+package org.partiql.lang.eval
+
+import org.junit.jupiter.api.Test
+import org.partiql.lang.eval.evaluatortestframework.EvaluatorTestCase
+
+class EvaluatingCompilerGraphMatchTests : EvaluatorTestBase() {
+
+    private val session = sessionOf(
+        graphs = mapOf(
+            "g1" to """{ nodes: [ {id: n1} ], edges: [] }"""
+        )
+    )
+
+    @Test
+    fun testG1() {
+        runEvaluatorTestCase(
+            EvaluatorTestCase(
+                query = "g1",
+                expectedResult = "g1" // cheating, but the lookup in the session should be happening
```

Comment:
I think your point is correct.  
However, I moved and enhanced this into GraphsInSessionTests.kt in the downstream #1074, so I'll try to fix it there, into something like `assertDoesNotThrow`.

Reply:
Upon looking into this, the `runEvaluatorTestCase` methods that we have in the harness do not offer something similar to `assertDoesNotThrow` and it does not seem worth it introducing just for the purpose here. 
As it evolved in PR #1074 in `GraphsInSessionTests`, it can also be said to test for various methods of defining graphs and exercising the sample graphs that got checked in.  
So, please see and comment there if there are more concerns.